src/modules/cmd_duration.rs

```diff
@@ -86,6 +82,43 @@ fn render_time_component((component, suffix): (&u128, &&str)) -> String {
     }
 }
 
+#[cfg(not(feature = "notify-rust"))]
+fn undistract_me(module: Module, _elapsed: u128) -> Module {
+    let config: CmdDurationConfig = CmdDurationConfig::try_load(module.config);
+    if config.show_notifications {
+        log::debug!("This version of starship was built without notification support.");
+    }
+
+    module
+}
+
+#[cfg(feature = "notify-rust")]
+fn undistract_me(module: Module, elapsed: u128) -> Module {
+    use ansi_term::{unstyle, ANSIStrings};
+    use notify_rust::{Notification, Timeout};
+
+    let config: CmdDurationConfig = CmdDurationConfig::try_load(module.config);
```

Comment:
I think passing the config into this function would be better since we wouldn't have to load it twice.